examples/stable_diffusion_v2/inference/sd_infer.py

```diff
@@ -168,6 +169,22 @@ def main(args):
                 )
             detected_map = apply_segment(img)
             detected_map = cv2.resize(detected_map, (W, H), interpolation=cv2.INTER_NEAREST)
+        elif args.controlnet_mode == "openpose":
+            if args.control_path is not None:
+                detected_map = cv2.imread(args.control_path)
+                print("D---: use input openpose image: ", args.control_path)
+            else:
+                if os.path.exists(args.inputs.condition_ckpt_path):
+                    apply_segment = OpenposeDetector(annotator_ckpts_path=args.inputs.condition_ckpt_path)
+                else:
+                    apply_openpose = OpenposeDetector()
+
+                # cong TODO: make sure the resolution is correct
+                # resize_image(input_image, detect_resolution)
+                detected_map, _ = apply_openpose(img)
+                detected_map = HWC3(detected_map)
+            print("D--: sum openpose: ", (detected_map / 255.0).sum())
```

Comment:
Is this printing for debugging only? Should be deleted before merging.

Reply:
yes thanks for reminding